### PR TITLE
Add OPUS Books Dataset

### PR DESCRIPTION
**Pull request by abhishekkrthakur**

### CODE REVIEW

**Review by lhoestq — COMMENTED** [review]
can you rename the dataset to something less generic ?

maybe `opus_books` ?

File: datasets/books/books.py [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+                x = x.strip()
+                y = y.strip()
+                result = (
+                    sentence_counter,
+                    {
+                        "id": str(sentence_counter),
+                        "translation": {l1: x, l2: y},
+                    },
+                )
+                sentence_counter += 1
```
Comment (lhoestq) [inline datasets/books/books.py:187]:
```suggestion
```

**Comment by abhishekkrthakur** [comment]
@lhoestq  done

**Review by lhoestq — APPROVED** [review]
Awesome thanks !

**Pull request merged**